shared/src/main/kotlin/com/egm/stellio/shared/model/EntitiesEvent.kt

```diff
@@ -0,0 +1,98 @@
+package com.egm.stellio.shared.model
+
+import java.net.URI
+
+sealed class EntitiesEvent(
```

Comment:
This class has a lot of nullable values which, in my opinion, misses the objective to have a clear contract for each type of event. In other words, when I get a `EntityCreateEvent`, I want an object that has only the expected mandatory attributes for an event of this type (and not all the possible attributes with the majority of them being null).

Reply:
I would rename this class EntityEvent and the kt file EntityEvents

Reply:
To not break the existing code, I choosed EntitiesEvent.
I'll rename it to EntityEvent once it's fully integrated

Reply:
OK, BTW it's "chose"